perf_test/sparse/KokkosSparse_spmv.cpp

```diff
@@ -125,6 +130,11 @@ void matvec(AType& A, XType x, YType y, Ordinal rows_per_thread, int team_size,
         case CUSPARSE:
                 cusparse_matvec(A, x, y);
```

Comment:
It would be nice if these calls used the unification layer and Controls "algorithm" at some point, but this is fine for now (and anyway, it's consistent with cusparse_matvec).

Reply:
Yeah, I was pretty annoyed when I saw how the test handles the cusparse handle and matrix descriptor.
I want to overhaul that so that it at leasts uses the `Controls` object and as pointed out, at the point we can also just have it use the unification layer.